primitives/src/merkle_tree/field_based_mht/parameters/tweedle_dee.rs

```diff
@@ -0,0 +1,104 @@
+use algebra::{
+    fields::tweedle::Fr as TweedleFr,
+    biginteger::BigInteger256,
+    field_new,
+};
+
+use crate::{
+    crh::poseidon::TweedleFrPoseidonHash,
+    FieldBasedMerkleTreePrecomputedEmptyConstants,
+};
+
+// PoseidonHash("This represents an empty Merkle Root for a TweedleDeePoseidonHash based Merkle Tree.")
+pub const TWEEDLE_DEE_PHANTOM_MERKLE_ROOT: TweedleFr =
+    field_new!(TweedleFr, BigInteger256([
+        3037328570344736808,
+        7713672412249441083,
+        9580110273020338221,
+        816345156093001009
+    ]));
+
+pub const TWEEDLE_DEE_MHT_POSEIDON_PARAMETERS: FieldBasedMerkleTreePrecomputedEmptyConstants<'static, TweedleFrPoseidonHash> =
+    FieldBasedMerkleTreePrecomputedEmptyConstants {
```

Comment:
From tree perspective "zero" nodes are not empty but just zero (the fact we precompute the above hashes values is another story). Rename from *Empty* to *Zero*